magefiles/lint.go

```diff
@@ -89,7 +89,7 @@ func (Lint) Analyzers() error {
 		"-closeafterusagecheck.must-be-closed-after-usage-types=github.com/authzed/spicedb/pkg/datastore.RelationshipIterator",
 		"-closeafterusagecheck.skip-pkg=github.com/authzed/spicedb/pkg/datastore,github.com/authzed/spicedb/internal/datastore,github.com/authzed/spicedb/internal/testfixtures",
 		"-paniccheck",
-		"-paniccheck.skip-files=_test,zz_",
+		"-paniccheck.skip-files=_test,zz_,profiling.go",
```

Comment:
I'd remove this and just name the functions `mustWhatever`, to allow them to bypass